### PR TITLE
Rancher k8s upgrade: K8s v1.24 & Rancher v2.6.9

### PR DESCRIPTION
**Pull request by pranavv0**
Updated Rancher Version to 2.6.9
Changes as per K8s v1.24
> Updated HPA to autoscale/v2
> Updated Cron to batch/v2

### CODE REVIEW

**Comment by abhi4578** [comment]
Upgrade the cloud init scripts with new rancher version @pranavv0

**Comment by abhi4578** [comment]
@pranavv0 also upgrade cloud init scripts in aws

**Comment by abhi4578** [comment]
Tasks remaining @pranavv0 
- [ ] Use azure file csi instead of azurefile in-tree as its depreciated
- [ ] Add ebs-csi-storage-class in example aws resource values and remove ebs-storage-class as its depreciated
- [ ] Upgrade Nginx ingress as it uses depreciated PSP , PDB
- [ ] Upgrade cloud init scripts in aws

**Comment by abhi4578** [comment]
@pranavv0 please merge the master branch to ur local branch and resolve conflicts. Also please cherry pick commits and pull another PR at 4.0.0 branch

**Comment by abhi4578** [comment]
We no longer using databroker manifest files, use helm files

**Review by karun-singh — APPROVED** [review]

File: K8s-deployment/K8s-cluster/addons/storage/azure/README.md [hunk 1]
```diff
@@ -43,13 +43,55 @@ kubectl create -f azure-cloud-provider.yaml
 ``` kubectl apply -f azuredisk-storage-class.yaml```
 
 # Deployment with AzureFile
-1. Create a storage account on azure for the shared file system. Follow the documentation [here](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal).
-2. Update the `azurefile-storageclass.yaml` with the created storage account name.
-3. Following command will deploy the storageclass
+For in-tree azuredisk drivers, ```kubernetes.io/azure-file``` provisioner can be used in the [azurefile-storage-class.yaml](./azurefile-storage-class.yaml).
```
Comment (abhi4578) [inline K8s-deployment/K8s-cluster/addons/storage/azure/README.md:46]: Document azurefile csi drivers
- Minimum k8s version - 1.21+
- Pre-requisites
- Installation of csi drivers through helm chart with commands 
and ref: https://github.com/kubernetes-sigs/azurefile-csi-driver#install-driver-on-a-kubernetes-cluster

File: K8s-deployment/K8s-cluster/addons/storage/azure/README.md [hunk 2]
```diff
@@ -43,13 +43,55 @@ kubectl create -f azure-cloud-provider.yaml
...
+1. Following command will deploy the storageclass
 ``` kubectl apply -f azurefile-storage-class.yaml```
 
-## Links
+## Deployment with AzureDisk csi drivers
```
Comment (abhi4578) [inline K8s-deployment/K8s-cluster/addons/storage/azure/README.md:50]: This section is repeated, its documented before in same readme

**Review by abhi4578 — CHANGES_REQUESTED** [review]

**Pull request merged**